qemu/tests/cfg/edk2_basic.cfg

```diff
@@ -0,0 +1,32 @@
+- edk2_basic:
+    virt_test_type = qemu
+    only Linux
+    only x86_64, aarch64
```

Comment:
```suggestion
    only x86_64, aarch64
    x86_64:
        only ovmf
```

Reply:
Oh sorry I didn't understand your suggestion correctly before, many thanks for the example, I have corrected it.